pokemon_v2/tests.py

```diff
@@ -1275,6 +1274,17 @@ def setup_characteristic_description_data(
 
         return characteristic_description
 
+    @classmethod
+    def trophy_garden_special_encounter_data(
```

Comment:
Please rename to be consistent with other setup function names
```suggestion
    def setup_trophy_garden_special_encounter_data(
```